src/applications/vaos/appointment-list/pages/AppointmentsPage/RequestListItem.jsx

```diff
@@ -35,12 +33,7 @@ export default function RequestListItem({ appointment, facility }) {
   const preferredDate = appointment?.preferredDate;
   const idClickable = `id-${appointment.id?.replace('.', '\\.')}`;
 
-  const featureBreadcrumbUrlUpdate = useSelector(state =>
-    selectFeatureBreadcrumbUrlUpdate(state),
-  );
-  const link = `${featureBreadcrumbUrlUpdate ? 'pending' : 'requests'}/${
-    appointment.id
-  }`;
+  const link = `${'requests'}/${appointment.id}`;
```

Comment:
```suggestion
  const link = `${'pending'}/${appointment.id}`;
```

Seems like this should be pending instead of requests